src/Symfony/Component/Notifier/Bridge/Twilio/TwilioTransport.php

```diff
@@ -58,20 +59,29 @@ protected function doSend(MessageInterface $message): SentMessage
             throw new UnsupportedMessageTypeException(__CLASS__, SmsMessage::class, $message);
```

Comment:
`supports()` method in line 53 needs to check for the supported `TwilioOptions` class:

```php
    public function supports(MessageInterface $message): bool
    {
        return $message instanceof SmsMessage && (null === $message->getOptions() || $message->getOptions() instanceof TwilioOptions);
    }
```